packages/auth/__tests__/auth.test.ts

```diff
@@ -43,4 +43,29 @@ describe('Auth', function () {
       expect(bar).toEqual(['10.0.2.2', 9099]);
     });
   });
+
+  describe('tenantId', function () {
+    it('should can set tenantId ', function () {
+      const auth = firebase.app().auth();
+      auth.setTenantId('test-id').then(() => {
+        expect(auth.tenantId).toBe('test-id');
+      });
+    });
+    it('should throw error when set tenantId to null ', function () {
```

Comment:
```suggestion
    it('should throw error if tenantId is null ', function () {
```

Reply:
Thanks for typescript, we don't allow to call this method with  null. so no test for this scenario any more, deleted it.